### ISSUE TITLE
Misplaced js1.dll in EventStore.Client.Embedded 3.2.0 NuGet package

### ISSUE THREAD

**Issue by ArturDorochowicz** [EventStore/EventStore#666]
The EventStore.Client.Embedded 3.2.0 NuGet package doesn't work.

```.targets``` file references this: 
```
    <Js1DllPath>$(MSBuildThisFileDirectory)..\native\x64\win\js1.dll</Js1DllPath>
```
but the file is at
```
native\x64\js1.dll
```

Note the missing ```win``` directory.

**Comment by jen20** [comment]
This is a bug, we'll fix it and get patch out shortly.

**Comment by jen20** [comment]
Fixed in Event Store 3.2.1
